__tests__/app.test.js

```diff
@@ -45,24 +56,33 @@ describe("GET /api/categories", () => {
   });
 });
 
-describe("GET /api/reviews:review_id", () => {
+describe("GET /api/reviews/:review_id", () => {
   it("200: Returns a review object with the correct properties.", () => {
     return request(app)
       .get("/api/reviews/1")
       .expect(200)
       .then(({ body }) => {
         const { reviewObj } = body;
         expect(reviewObj).toMatchObject({
-          review_id: expect.any(Number),
-          title: expect.any(String),
-          review_body: expect.any(String),
-          designer: expect.any(String),
-          review_img_url: expect.any(String),
-          votes: expect.any(Number),
-          category: expect.any(String),
-          owner: expect.any(String),
-          created_at: expect.any(String),
+          review_id: 1,
+          title: "Agricola",
+          review_body: "Farmyard fun!",
+          designer: "Uwe Rosenberg",
+          review_img_url:
+            "https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700",
+          votes: 1,
+          category: "euro game",
+          owner: "mallionaire",
+          created_at: "2021-01-18T10:00:20.514Z",
         });
       });
   });
+  it("400: should return bad request when endpoint has an id which does not exist.", () => {
```

Comment:
Nitpicky from me, but I would change this test description to mention it's an invalid id!